src/type_sys/type_system.rs

```diff
@@ -5,8 +5,8 @@ use freight_vm::TypeSystem;
 pub struct FenderTypeSystem;
 
 impl TypeSystem for FenderTypeSystem {
-    type V = FenderReference;
-    type B = FenderBinaryOperator;
-    type U = FenderUnaryOperator;
-    type T = TypeId;
+    type Value = FenderReference;
+    type BinaryOp = FenderBinaryOperator;
+    type UnaryOp = FenderUnaryOperator;
+    type TypeId = TypeId;
```

Comment:
I would like to change one of these names cause `TypeId` gets used in relatively the same context and makes it very hard to tell the two apart.